packages/amazon-ssm-agent/amazon-ssm-agent.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=amazon-ssm-agent
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/amazon-ssm-agent/amazon-ssm-agent
```

Comment:
You should install the binary at `/usr/bin` without the `amazon-ssm-agent`

Reply:
Yes, that's what I did originally. But while testing, I found that if I did this, I was not able to SSM into the instance.

However, this was with host-containers enabled, which we determined later is the incorrect setup, so let me try again.